modules/restoring-central-database.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * cli/managing-central-database.adoc
+:_content-type: PROCEDURE
+[id="restoring-central-database_{context}"]
+= Restoring Central database
+
+.Procedure
+
+* Run the following command to restore Central's database:
```

Comment:
For consistency, need to mention the `ROX_ENDPOINT` here as well.

Reply:
I have created a "Prerequisites" section under "Managing Central database"